src/webgpu/util/device_pool.ts

```diff
@@ -243,10 +250,11 @@ type DeviceHolderState = 'free' | 'reserved' | 'acquired';
 class DeviceHolder implements DeviceProvider {
   readonly device: GPUDevice;
   state: DeviceHolderState = 'free';
-  lostReason?: string; // initially undefined; becomes set when the device is lost
+  // initially undefined; becomes set when the device is lost
+  lost?: GPUDeviceLostInfo;
```

Comment:
nit: maybe `lostInfo` just to disambiguate from `GPUDevice.lost` which is a promise

Reply:
Done